backend/src/environment.ts

```diff
@@ -21,31 +12,40 @@ export class Environment {
   isArmLinux: boolean
   isPi: boolean
   isPiZero: boolean
-  platform: string
   simpleOS: Ios
   userPath: string
   adminPath: string
   binPath: string
+  deprecatedBinaries: string[]
 
   constructor() {
+    const elevated: boolean = true //this.isElevated - always elevated for now
+
+    // @ts-ignore
```

Comment:
Why ignoring? Underscore variable name? If so, try `process.config.variables['arm_version']`

Reply:
I did a lot of research as to how to do this, but typescript still gives a warning:
> Property 'arm_version' does not exist on type '{ clang: number; host_arch: string; node_install_npm: boolean; node_install_waf: boolean; node_prefix: string; node_shared_openssl: boolean; node_shared_v8: boolean; node_shared_zlib: boolean; ... 6 more ...; visibility: string; }'.ts(2339)